apps/api/src/domain/services/query-services/base-query.service.ts

```diff
@@ -0,0 +1,127 @@
+import {
```

Comment:
I thought that an abstract class didn't have implementation details, but `fetchById` etc. seem to return values etc.  Still need to get a handle on what an abstract class is...

Reply:
See `pro-d` channel for a discussion.